core/src/main/java/org/apache/accumulo/core/client/ClientSideIteratorScanner.java

```diff
@@ -213,7 +213,7 @@ public ClientSideIteratorScanner(final Scanner scanner) {
     smi = new ScannerTranslatorImpl(scanner, scanner.getSamplerConfiguration());
     this.range = scanner.getRange();
     this.size = scanner.getBatchSize();
-    this.timeOut = scanner.getTimeout(MILLISECONDS);
+    this.retryTimeout = scanner.getTimeout(MILLISECONDS);
     this.batchTimeOut = scanner.getTimeout(MILLISECONDS);
```

Comment:
Does it make sense to also update `batchTimeOut` to match the case of `retryTimeout` for uniformity?

Reply:
It appears that the use of timeout between Scanner and BatchScanner is inconsistent. Even though they use the same API method the BatchScanner actually times out. This is an issue we should fix. This PR is step 1 I think.